src/main/java/nextstep/subway/path/domain/Path.java

```diff
@@ -24,4 +35,17 @@ public static Path of(List<Station> stations, Integer totalDistance) {
     public Integer getTotalDistance() {
         return totalDistance;
     }
+
+    public Integer getTotalFare() {
+        return totalFare.get();
+    }
+
+    private static Fare getAdditionalFare(List<Line> includedLines) {
+        return includedLines.stream()
+            .map(it -> it.getAdditionalFare())
+            .sorted(Comparator.comparingInt(Fare::get).reversed())
+            .findFirst()
+            .orElse(Fare.valueOf(0));
```

Comment:
stream의 max를 이용해볼 수도 있겠네요 ! 그리고 Faire는 요금을 의미하는 값 객체이기 때문에 Comparable 인터페이스를 구현하는 것도 고민해볼 수 있을 것 같습니다 :)

```suggestion
        return includedLines.stream()
                .map(Line::getAdditionalFare)
                .max(Comparator.comparingInt(Fare::get))
                .orElse(Fare.valueOf(0));
```